src/app/components/AddDog/AddDog.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import DogIcon from "../Icons/DogIcon";
+import PlusIcon from "../Icons/PlusIcon";
+import styles from "./AddDog.module.css";
+
+type AddDogProps = {
+  label: string;
+};
+
+export default function AddDog({ label }: AddDogProps): JSX.Element {
+  return (
+    <div>
+      <label className={styles.addDog__label}>
+        {label}
+        <button className={styles.addDog__button}>
+          <div className={styles.addDog__icons}>
```

Comment:
I am pretty sure that you don't need this `div`, instead, style the button to align your icons.

Reply:
Deleted the div and changed styling.